### PR TITLE
refactor: reorganize and improve PieChart, Placeholder and SelectNumber stories

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Refactor

#### What is expected?

reorganize and improve PieChart, Placeholder and SelectNumber stories

### CODE REVIEW

**Review by matthprost — CHANGES_REQUESTED** [review]

File: src/components/SelectNumber/__stories__/index.stories.tsx [hunk 1]
```diff
@@ -1,152 +1,29 @@
...
 export default {
   component: SelectNumber,
-  title: 'Components/Data Display/SelectNumber',
   decorators: [
     StoryComponent => (
-      <StyledDiv>
+      <div
+        style={{
+          marginBottom: '1em',
+          marginLeft: '2em',
```
[the remark this answers is not in the document]
Comment (baptisteArno) [reply src/components/SelectNumber/__stories__/index.stories.tsx:11]: My bad, that's a mistake

[the remark this answers is not in the document]
Comment (vincentaudebert) [reply src/components/SelectNumber/__stories__/index.stories.tsx:11]: What's inside Menu stories?

File: src/components/PieChart/__stories__/Template.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { ComponentStory } from '@storybook/react'
+import PieChart from '..'
+import { data } from './mockData'
+
+export const Template: ComponentStory<typeof PieChart> = ({ ...props }) => (
+  <PieChart data={data} {...props} />
+)
```
Comment (baptisteArno) [inline src/components/PieChart/__stories__/Template.stories.tsx:7]:
```suggestion
export const Template: ComponentStory<typeof PieChart> = (props) => (
  <PieChart data={data} {...props} />
)
```

File: src/components/SelectNumber/__stories__/Template.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  return (
+    <SelectNumber
+      minValue={0}
+      onChange={val => typeof val === 'number' && setValue(val)}
+      value={value}
+      {...props}
+    />
+  )
+}
```
Comment (baptisteArno) [inline src/components/SelectNumber/__stories__/Template.stories.tsx:16]: Template should be:

```tsx
export const Template: ComponentStory<typeof SelectNumber> = (props) => (
    <SelectNumber {...props} />
  )
```

**Review by matthprost — APPROVED** [review]

**Review by baptisteArno — APPROVED** [review]

**Pull request merged**